strawberry/object_type.py

```diff
@@ -117,6 +118,7 @@ def _process_type(
         origin=cls,
         extend=extend,
         _fields=fields,
+        is_type_of=is_type_of,
```

Comment:
I think we should do this for pydantic too :)

Reply:
Done in [`fdc290a` (#1406)](https://github.com/strawberry-graphql/strawberry/pull/1406/commits/fdc290a53088969581c7dffd8313e57bf3247ef3)